jerry-core/parser/js/js-parser-expr.c

```diff
@@ -2619,26 +2619,25 @@ parser_process_unary_expression (parser_context_t *context_p, /**< context */
             parser_emit_cbc (context_p, (uint16_t) (CBC_CALL0 + (call_arguments * 6)));
             continue;
           }
-          if (opcode == CBC_CALL_PROP)
+          else if (opcode == CBC_CALL_PROP)
           {
             parser_emit_cbc (context_p, (uint16_t) (CBC_CALL0_PROP + (call_arguments * 6)));
             continue;
           }
-          if (opcode == CBC_NEW)
+          else if (opcode == CBC_NEW)
           {
             parser_emit_cbc (context_p, (uint16_t) (CBC_NEW0 + call_arguments));
             continue;
           }
         }
-
-        if (call_arguments == 2)
+        else if (call_arguments == 2)
```

Comment:
@zherczeg 

I think this is worth only if `call_arguments <= 1` and opcode is `CBC_EXT_SUPER_CALL`. But the others, changes on opcode arms have no effect as you said.

The commit message doesn't make sense with this, so if it is worth doing, I'll change the commit and ask for a review again or close it.

Reply:
Yes this one is a good change, worth a commit.

Reply:
Thank you for the review!